packages/psss-server/src/connections/MeditrakConnection.ts

```diff
@@ -46,17 +46,17 @@ export class MeditrakConnection extends ApiConnection {
     const results = (await this.get(`surveyResponses/`, {
       page: '0',
       pageSize: '1',
-      columns: `["entity.code","survey.code","submission_time","id"]`,
+      columns: `["entity.code","survey.code","data_time","id"]`,
       filter: JSON.stringify({
         'survey.code': { comparisonValue: surveyCode },
         'entity.code': { comparisonValue: orgUnitCode },
-        submission_time: {
+        data_time: {
           comparator: 'BETWEEN',
           comparisonValue: [startDate, endDate],
```

Comment:
`convertPeriodStringToDateRange` returns these values as date format `YYYY-MM-DD` so shouldn't need to have the timezone stripped